### PR TITLE
Outline of Level class

### PR DESCRIPTION
**Pull request by eelliott-hub**

### CODE REVIEW

File: platformer/src/processing/MissionPossible/Sprite.pde [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  float xAcceleration;
+  int spriteWidth;
+  int spriteHeight;
+  int spriteLayer;
+  int health; // Move to PlayerControlledSprite if only player has health
+  boolean onGround;
+  PImage image;
+  // TODO direction - type?
+  // TODO damage - type?
+  // TODO collision - type?
```
Comment (cej-harris) [inline platformer/src/processing/MissionPossible/Sprite.pde:15]: Having thought about it a bit more, given that we have x and y Acceleration, probably makes sense to remove `direction` and have `xSpeed` and `ySpeed` instead?
`damage` would probably be a float, `collision` a boolean?

File: platformer/src/main/java/Level.java
Comment (cej-harris) [inline platformer/src/main/java/Level.java:1]: Given the Processing refactor, probably worth removing this file

File: platformer/src/main/resources/level1.json
Comment (cej-harris) [inline platformer/src/main/resources/level1.json:1]: Probably makes sense moving this file to another directory

File: platformer/src/main/resources/level1.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+   "position": {
+     "x": 372,
+     "y": 137
+   },
+   "spriteWidth": 20,
+   "spriteHeight": 20,
+   "layer": 1
+   }
+ ]
+ }
```
Comment (cej-harris) [inline platformer/src/main/resources/level1.json:27]:
```suggestion
 }
 
```

Git likes files ending with an empty line

**Review by cej-harris — CHANGES_REQUESTED** [review]

File: platformer/src/processing/MissionPossible/Sprite.pde (hunk 1 above)
Comment (eelliott-hub) [reply platformer/src/processing/MissionPossible/Sprite.pde:15]: Have made these changes - probably need to think about whether we need all of lives, health and damage.

**Review by cej-harris — APPROVED** [review]
LGTM

**Pull request merged**